extensionsGallery-insider.json

```diff
@@ -64,14 +64,14 @@
 				},
 				"versions": [
 					{
-						"version": "0.1.0",
-						"lastUpdated": "1/1/2018",
+						"version": "0.1.1",
+						"lastUpdated": "4/24/2018",
 						"assetUri": "",
 						"fallbackAssetUri": "fallbackAssetUri",
 							"files": [
 							{
 								"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
-								"source": "https://sqlopsextensions.blob.core.windows.net/march-extensions/whoisactive/whoisactive-0.1.0.vsix"
+								"source": "https://sqlopsextensions.blob.core.windows.net/extensions/ServerReports/server-report-0.1.1.vsix"
```

Comment:
This is in the sp_whoisactive extension section, server reports is the next one. Did you mean to update that entry instead?

Reply:
good call!

---
In reply to: [183906479](https://github.com/Microsoft/sqlopsstudio/pull/1254#discussion_r183906479) [](ancestors = 183906479)

Reply:
oops 😄